src/types/index.ts

```diff
@@ -31,6 +32,7 @@ export interface TokenBalanceDetails extends TokenDetails {
 
 export interface TokenList {
   getTokens: (networkId: number) => TokenDetails[]
+  fetchTokenIdsFromExchange: (networkId: number) => Promise<void>
```

Comment:
With this kind of name I'd expect this function to return an array of you know what.

Reply:
What name would you suggest instead?

Reply:
Renamed to `updateTokenIdsForNetwork`. Let me know whether that's better.

Reply:
:+1: